packages/upswyng-web/src/components/__tests__/Resource.test.tsx

```diff
@@ -25,27 +32,18 @@ jest.mock("../../App.styles", () => ({
     },
   },
 }));
-jest.mock("react-router-dom", () => ({
-  useParams: () => ({ resourceId: "some resource ID" }),
-}));
-jest.mock("../PageBanner", () => "PageBanner");
-jest.mock("../Schedule", () => "Schedule");
-jest.mock("../Map", () => "Map");
-jest.mock("../useResource.tsx");
-const mockedUseResource = useResource as jest.Mock;
 
-describe("<Resource/>", () => {
-  mockedUseResource.mockImplementation(() => foodResource) as typeof jest.mock;
-  const wrapper = shallow(<Resource />);
+const mockUseResource = useResource as jest.Mock;
 
-  it("renders the charityname property of the resource prop object", () => {
-    expect(wrapper.find("PageBanner").prop("text")).toBe(foodResource.name);
-  });
+describe("<Resource/>", () => {
+  const setup = (overrides = {}) => {
+    return mount(<Resource {...overrides} />);
+  };
```

Comment:
Created a quick setup function that can be used to override any props we want to provide an individual render of `<Resource/>`. If we add any tests in the future that are prop-based, we can use the setup like so:

```js
it('renders lollipops equal to the passed number of lollipops', () => {
   const numLollipops = 2;
   const wrapper = renderWith({ numLollipops });
   expect(wrapper.find('lollipop')).toHaveLength(numLollipops);
})
```

so doing `renderWith({ numLollipop: 2 })` will essentially render the resource component with a `lollipop` prop that equals `2`: `<Resource numLollipops={2} />`

Reply:
Nice 😎